Mathlib/NumberTheory/NumberField/InfinitePlace/TotallyRealComplex.lean

```diff
@@ -146,12 +146,18 @@ A number field `K` is totally complex if all of its infinite places are complex.
 @[mk_iff] class IsTotallyComplex (K : Type*) [Field K] [NumberField K] where
   isComplex : ∀ v : InfinitePlace K, v.IsComplex
 
-variable {K : Type*} [Field K] [NumberField K]
+variable {F : Type*} [Field F] {K : Type*} [Field K] [NumberField K] [Algebra F K]
 
 theorem nrRealPlaces_eq_zero_iff :
     nrRealPlaces K = 0 ↔ IsTotallyComplex K := by
   simp [Fintype.card_eq_zero_iff, isEmpty_subtype, isTotallyComplex_iff]
 
+theorem IsTotallyComplex.isConj_ne_one [IsTotallyComplex K] {φ : K →+* ℂ} {σ : K ≃ₐ[F] K}
+    (hφ : ComplexEmbedding.IsConj φ σ) : σ ≠ 1 := by
+  by_contra h
```

Comment:
```suggestion
  intro h
```
It does not make any difference, but you're really not proving it "by contradiction", you're proving a `not` statement using the definition, no?